website/content/v1.5/advanced/metal-network-configuration.md

```diff
@@ -0,0 +1,430 @@
+it doesn't support all kinds of network configuration.
+
+## Network Configuration Format
+
+Talos `META`-based network configuration is a YAML file with the following format:
+
+```yaml
+addresses:
+    - address: 147.75.61.43/31
+      linkName: bond0
+      family: inet4
+      scope: global
+      flags: permanent
+      layer: platform
+    - address: 2604:1380:45f2:6c00::1/127
+      linkName: bond0
+      family: inet6
+      scope: global
+      flags: permanent
+      layer: platform
+    - address: 10.68.182.1/31
+      linkName: bond0
+      family: inet4
+      scope: global
+      flags: permanent
+      layer: platform
+links:
+    - name: eth0
+      up: true
+      masterName: bond0
+      slaveIndex: 0
+      layer: platform
+    - name: eth1
+      up: true
+      masterName: bond0
+      slaveIndex: 1
+      layer: platform
+    - name: bond0
+      logical: true
+      up: true
+      mtu: 0
+      kind: bond
+      type: ether
+      bondMaster:
+        mode: 802.3ad
+        xmitHashPolicy: layer3+4
+        lacpRate: slow
+        arpValidate: none
+        arpAllTargets: any
+        primaryReselect: always
+        failOverMac: 0
+        miimon: 100
+        updelay: 200
+        downdelay: 200
+        resendIgmp: 1
+        lpInterval: 1
+        packetsPerSlave: 1
+        numPeerNotif: 1
+        tlbLogicalLb: 1
+        adActorSysPrio: 65535
+      layer: platform
+routes:
+    - family: inet4
+      dst: ""
+      src: ""
+      gateway: 147.75.61.42
+      outLinkName: bond0
+      table: main
+      priority: 1024
+      scope: global
+      type: unicast
+      flags: ""
+      protocol: static
+      layer: platform
+    - family: inet6
+      dst: ""
+      src: ""
+      gateway: '2604:1380:45f2:6c00::'
+      outLinkName: bond0
+      table: main
+      priority: 2048
+      scope: global
+      type: unicast
+      flags: ""
+      protocol: static
+      layer: platform
+    - family: inet4
+      dst: 10.0.0.0/8
+      src: ""
+      gateway: 10.68.182.0
+      outLinkName: bond0
+      table: main
+      scope: global
+      type: unicast
+      flags: ""
+      protocol: static
+      layer: platform
+hostnames:
+    - hostname: ci-blue-worker-amd64-2
+      domainname: ""
+      layer: platform
+resolvers: []
+timeServers: []
+```
+
+Every section is optional, so you can configure only the parts you need.
+The format of each section matches the respective network [`*Spec` resource](<{{ relref "../learn-more/networking-resources" }}>) `.spec` part, e.g the `addresses:`
+section matches the `.spec` of `AddressSpec` resource:
+
+```yaml
+# talosctl get addressspecs bond0/10.68.182.1/31 -o yaml | yq .spec                                                                                                                                                                                                                        1471ms  Thu 28 Sep
```

Comment:
remove the timestamp

Reply:
thanks, removed, I'm blind. I saw the line looked weird, but I didn't see the timestamp :-D